pom.xml

```diff
@@ -25,11 +25,12 @@
         <api-helper-java.version>1.1.0-rc1</api-helper-java.version>
         <rest-service-common-library.version>0.0.3</rest-service-common-library.version>
         <environment-reader-library.version>1.3.2</environment-reader-library.version>
+        <jackson.version>2.15.3</jackson.version>
         <!-- Logging -->
         <log4j-bom.version>2.17.0</log4j-bom.version>
         <structured-logging.version>1.9.12</structured-logging.version>
-        <avro.version>1.8.1</avro.version>
-        <ch-kafka.version>1.4.8</ch-kafka.version>
+        <avro.version>1.11.3</avro.version>
+        <ch-kafka.version>1.4.4</ch-kafka.version>
```

Comment:
does ch-kafka need to be dropped back from 1.4.8 to 1.4.4 ?  I thought this was okay?

Reply:
Actually we needed newer versions because of HIGH vuln. in avro 1.8.1